frontend/rust-lib/flowy-document2/src/parser/handler.rs

```diff
@@ -0,0 +1,34 @@
+use crate::manager::DocumentManager;
+use crate::parser::document_data_parser::DocumentDataParser;
+use crate::parser::parser_entities::{CopyParams, CopyPayloadPB, CopyResponsePB};
+use crate::utils::upgrade_document;
+use flowy_error::FlowyError;
+use lib_dispatch::prelude::{data_result_ok, AFPluginData, AFPluginState, DataResult};
+use std::sync::{Arc, Weak};
+
+/**
+  * copy handler
+    * @param data: AFPluginData<[CopyPayloadPB]>
+    * @param manager: AFPluginState<Weak<DocumentManager>>
+    * @return DataResult<[CopyResponsePB], FlowyError>
+ */
+pub(crate) async fn copy_handler(
```

Comment:
no need to create a new file. Just put this function to event_handler

Reply:
There are not only two methods for copy/paste, but also some additional covert methods are provided